staker-rewards/rewards.js

```diff
@@ -117,7 +121,17 @@ async function run() {
            )}
           `.replace(/\n/g, "\t")
       )
-    )
+
+      if (!operatorRewards.totalRewards.isZero()) {
+        rewards[
+          operatorRewards.operator
+        ] = operatorRewards.totalRewards
+          .integerValue(BigNumber.ROUND_DOWN)
+          .toString(16) // convert BN to hex
```

Comment:
And we write them in hex because otherwise, merkle generator has problems with precision? If so, let's make sure to add a comment about it.

Reply:
Done in 35f79f9